core/src/main/java/org/apache/sdap/mudrod/ontology/process/OntologyLinkCal.java

```diff
@@ -91,6 +91,7 @@ public Object execute() {
       e.printStackTrace();
     } catch (InterruptedException e) {
       e.printStackTrace();
```

Comment:
We should not be printing these stack traces, we should be logging them or re-throwing the exceptions.